frontend/src/pages/NotFoundPage/NotFoundPage.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable react/no-unescaped-entities */
```

Comment:
like this one, and I think I saw a couple more

Reply:
Hey @LoTerence, I'm prepping for an interview so I'll get to this probably this weekend.

Reply:
Hey @LoTerence, I'm prepping for an interview so I'll get to this probably this weekend.

Reply:
No problem, good luck on your interview!

Reply:
Hey @LoTerence, apologies for being MIA—I’ve been caught up with interviews. I added the plugins for hooks, accessibility, and the Tailwind classnames-order rule, and tested them all, so they’re working well. For the /* eslint-disable react/no-unescaped-entities */ question, I'm still investigating why the global rule in the ESLint config isn't taking effect. For now, adding the comment in the file bypasses the error successfully.

Since I’ve spent a bit of time on this, would you like me to close this as-is, or would you prefer I keep investigating a solution in the config? Let me know your thoughts!

Reply:
Great job thanks!

> Since I’ve spent a bit of time on this, would you like me to close this as-is, or would you prefer I keep investigating a solution in the config?
Ok thank you for investigating! No problem, we'll keep the comments there for now, and clean them up when we resolve the rest of the linter errors later.

> I also wrote some docs on the configurations and rules I added to the linter file.
Awesome! Thank you for doing this! I will review now.